src/doom/p_setup.c

```diff
@@ -389,6 +390,11 @@ void P_LoadThings (int lump)
 	P_SpawnMapThing(&spawnthing);
     }
 
+    if (!deathmatch)
```

Comment:
Though vanilla code is inconsistent, we would prefer that you put braces around all these `if` and `for` statements.

Reply:
Done.